src/lambdas/initiateAthenaQuery/handler.ts

```diff
@@ -22,6 +24,15 @@ export const handler = async (
 
   const zendeskId = retrieveZendeskIdFromEvent(event)
   appendZendeskIdToLogger(zendeskId)
+
+  if (zendeskId.startsWith('MR')) {
+    logger.warn('Manual query detected, no need to run athena query')
+    await publishToSNS(
```

Comment:
Could we create a really simple function to do the notification of data ready, something like `sendManualQueryDataReadyMessage`, put it in a separate file, and mock that out instead? That way, we keep the complications in the handler to an absolute minimum, and the tests end up being simpler too.